linkage/pvs_like_case_study/generate_simulated_data/generate_simulated_data_small_sample.ipynb

```diff
@@ -9,7 +9,8 @@
    "source": [
```

Comment:
<p>Code like this can guide us in enhancing the pseudopeople UX --- what functions would make it easy to explain to a novice user and/or make it easy for them/us to implement?  What would be well matched to a world where a lot of code is generated by AI assistants?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research_prl/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>